node/src/service.rs

```diff
@@ -24,17 +26,22 @@ use sc_service::{
     TFullClient,
     TaskManager,
 };
+use sc_client_api::ExecutorProvider;
+use sp_api::ConstructRuntimeApi;
 use sp_core::Pair;
 use sp_runtime::traits::BlakeTwo256;
 use sp_trie::PrefixedMemoryDB;
 use std::sync::Arc;
 use cumulus_primitives_core::ParaId;
-use sc_telemetry::{Telemetry, TelemetryWorker, TelemetryWorkerHandle};
-
+use sp_consensus::SlotData;
+use sc_telemetry::{Telemetry, TelemetryHandle, TelemetryWorker, TelemetryWorkerHandle};
+use substrate_prometheus_endpoint::Registry;
+use sp_keystore::SyncCryptoStorePtr;
+use sc_network::NetworkService;
 
 // Native executor instance.
 native_executor_instance!(
-    pub Executor,
+    pub DataHighwayRuntimeExecutor,
     datahighway_runtime::api::dispatch,
     datahighway_runtime::native_version,
```

Comment:
do we want benchmarking (i.e. https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/node/src/service.rs#L42)

Reply:
to be done in https://github.com/DataHighway-DHX/node/issues/228